wercker.yml

```diff
@@ -41,6 +41,15 @@ build:
           bin/cfmt
           git diff --exit-code
     - script:
+        name: gas
+        code: |
+          cd $GOPATH/src
+          git clone https://github.com/GoASTScanner/gas.git
+          cd gas
+          go install
+          cd $GOPATH/src/chain
+          gas
```

Comment:
When I run just `gas` locally, I get a usage message. Should this be like `gas chain/...` or something?